MySQL/connector.js

```diff
@@ -0,0 +1,28 @@
+const mysql = require('mysql2/promise');
+
+let logger = global.logger;
+
+module.exports = async (connectorData) => {
+	logger.trace("MySQL Connector: Invoked!");
+	logger.trace(`MySQL Connector: Connector Data: ${JSON.stringify(connectorData)}`);
+	logger.trace(`MySQL Connector: Connecting to MySQL: ${connectorData.database}`);
+	try {
+		const connection = await mysql.createConnection({
+			user: connectorData.user,
+			host: connectorData.host,
+			database: connectorData.database,
+			password: connectorData.password,
+			port: connectorData.port,
+		});
+		logger.info("MySQL Connector: Connected to MySQL");
+		logger.info(`MySQL Connector: DB set to ${connectorData.database}`);
+		return { MySQLCon: connection };
```

Comment:
Why reason why you are returning `return { MySQLCon: connection };` and not `return { connection };`?
IMHO, you should return `return { connection };`, so that it's consistent with other connectors.

Reply:
Scratch that!

You should return a client and db. Like this - `return {client, db}`. Just so that we are consistent with the way we return connection objects.

Reply:
Fixed.